precariobelasting/precariobelasting.json

```diff
@@ -0,0 +1,193 @@
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["schema", "id"],
+        "display": "id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "integer",
+            "description": "technische identificatie"
+          },
+          "geometry": {
+            "$ref": "https://geojson.org/schema/Geometry.json",
+            "description": "polygoon percariobelastinggebied"
+          },
+          "categorie": {
+            "type": "string",
+            "description": "categorie percariobelasting"
+          },
+          "jaar": {
+            "type": "string",
+            "description": "publicatie jaar percariobelasting"
+          },
+          "stadsdeel": {
+            "type": "string",
+            "description": "stadsdeel waarop tarieven van toepassing zijn"
+          },
+          "gebied": {
+            "type": "number",
+            "description": "percariobelastinggebied"
+          },
+          "tariefOverdektTerrasPerJaar": {
+            "type": "string",
+            "provenance": "overdektTerrasPerJaar"
+          },
+          "tariefOnoverdektTerrasPerZomerseizoen": {
+            "type": "string",
+            "provenance": "onoverdektTerrasPerZomerseizoen"
+          },
+          "tariefOnoverdektTerrasPerWinterseizoen": {
+            "type": "string",
+            "provenance": "onoverdektTerrasPerWinterseizoen"
+          }
+        }
+      }
+    },
+    {
+      "id": "bedrijfsvaartuigen",
+      "type": "table",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["schema", "id"],
+        "display": "id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "integer",
+            "description": "technische identificatie"
+          },
+          "geometry": {
+            "$ref": "https://geojson.org/schema/Geometry.json",
+            "description": "polygoon percariobelastinggebied"
+          },
+          "categorie": {
+            "type": "string",
+            "description": "categorie percariobelasting"
+          },
+          "jaar": {
+            "type": "string",
+            "description": "publicatie jaar percariobelasting"
+          },
+          "stadsdeel": {
+            "type": "string",
+            "description": "stadsdeel waarop tarieven van toepassing zijn"
+          },
+          "gebied": {
+            "type": "number",
+            "description": "percariobelastinggebied"
+          },
+          "tariefPerJaarPerM2": {
+            "type": "string"
```

Comment:
Hier kan ook een unit bij. Ben dan zelfs geneigd om het veld daarop aan te passen. En het is ook een decimal.

Dus 
`"tarief": { type: number, multipleOf: 0.01, unit: "[EUR]/a_g/m2"}`
moeten we wel even checken of dit dan ook al echt ondersteund wordt... pragmatisch gezien is de huidige kolomnaam houden voor vandaag wellicht nog wel wijzer:
`"tariefPerJaarPerM2": { type: number, multipleOf: 0.01, unit: "[EUR]/a_g/m2", title: "Tarief", "description": "Tarief in Euro per jaar per vierkante meter oppervlakte voortuig' }`

Reply:
Mooi, dit maakt het completer en beschrijvender